src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/DeleteTaskByIdControllerTest.java

```diff
@@ -39,57 +41,66 @@ public void setUp() {
 
     @Test
     public void should_succeed_when_terminate_reason_is_cancelled() {
+        boolean isFeatureEnabled = launchDarklyClient.getKey(
+            FeatureFlag.RELEASE_2_ENDPOINTS_FEATURE.getKey(),
+            authenticationHeaders.getValue(AUTHORIZATION));
+        if (!isFeatureEnabled) {
```

Comment:
One recommendation instead of making the call here:

We could maybe make it on the SpringBootFunctionalBaseTest this way all tests have access to this flag and it is called only once per run rather than per test, this should help in case your tests are running and someone toggles the flag mid-run, the tests would still pass but they might be testing different journeys if the flags are toggled mid-run